shared/components/GenericList/SearchInput.js

```diff
@@ -56,7 +56,13 @@ export function SearchInput({
         if (typeof entry === 'string') {
           if (entryMatchesSearch(entry)) return entry;
         }
-        return suggestionProperties?.map(properties => {
+        return suggestionProperties?.flatMap(properties => {
+          if (properties === 'metadata.labels' && entry.metadata?.labels) {
+            const labels = Object.keys(entry.metadata.labels).map(
+              key => `${key}=${entry.metadata.labels[key]}`,
+            );
```

Comment:
We could use `Object.entries` to make it nice `${key}=${value}`, but it's ok anyway.